### DISCUSSION TITLE
Increase group size to 50 members

### DISCUSSION THREAD

**Discussion by churik** [status-im/status-legacy#16632] [General]
# Feature Issue

### User Story

As a user, I want to be able to create group chat with more than 20 people 

### Description
Follow up on https://github.com/status-im/status-react/pull/13030

### Acceptance Criteria

- [ ] can create and message with 50 members
- [ ] all messages in group chat can be delivered to all members

**Comment by du82** [comment]
Ideally I would like to get it up to 100 users per group to compete with WeChat. For anything beyond 100 users, we could offer a prompt to move users to a community, and automatically move everyone, or send them all an invite.

**Comment by John-44** [comment]
Any increase needs to be tested and demonstrated to be performant on all Status platforms (as this limit needs to be exactly the same on all platform to ensure compatibility between problems).  Specifically:

- There used to be an ad-hoc group chat encryption performance issue on the Windows version of Status Desktop.  Before increasing the limit, we need to make sure we won't run into an issue again.

- We would also need to benchmark ad-hoc group chat scaling in Status Web.  Our web platform has a different stack from Status Mobile and Desktop (no status-go, everything in typescript on top of waku-js) so it might have different perf. issues from Status Mobile and Desktop.
